seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/config/PaimonConfig.java

```diff
@@ -106,6 +106,12 @@ public class PaimonConfig implements Serializable {
                     .withDescription(
                             "The specified loading path for the 'core-site.xml', 'hdfs-site.xml', 'hive-site.xml' files");
 
+    public static final Option<Map<String, String>> FS_CONF =
```

Comment:
What is the purpose of `paimon.fs.conf`  configuration?    Integrate `oss` `s3` configuration?

Reply:
yes, current connector don't write to s3/oss, I will add some examples

Reply:
Think you @yoogoc . The other filesystems are available as plugins in paimon, so Paimon-bundle has no s3 or oss dependencies. Other than that, I don't think we should have a separate fs.conf for all of this, so we can just do it in the original hadoop.conf. I already support writing to s3, it will be committed today, if you still need oss you can continue to develop base it.

Reply:
of course